.travis.yml

```diff
@@ -1,31 +1,12 @@
-
 language: python
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
-    #  Pypy versions
-  - "pypy"
-  - "pypy3"
-
-group: deprecated-2017Q4
-sudo: enabled
-
-services: 
-  - SQLite3
-  
-
-before_install: 
-  - "pip install pep8" 
-  - "pip install --upgrade setuptools"
-  - "pip install --upgrade pip"
-  
+before_install:  
+  - pip install --upgrade pip
 install:
-        - pip install -r requirements.txt
-        
-
+  - pip install flake8
 script: 
-        - py.test
-    
-
+  - flake8
```

Comment:
How about specifying the files explicitly?
Like this 
```
flake8 manager setup.py
```

Reply:
`flake8` already checks all available files with `.py` extension. then why do we need to specify files explicitly?

Reply:
@ganeshhubale it would be better if we keep it specific, later there can be some files for which we won't need to check PEP8. Simply we don't want flake8 to scan each and everything. Instead let's be specific by mentioning the package and file names.

Reply:
@bhavin192  yes, I got it. I will change as per you mentioned above.

Reply:
@bhavin192  Is it ok if changes as below:
script:
`flake8 manager/*.py`
`flake8 setup.py`

Reply:
@ganeshhubale please check my first comment

Reply:
@bhavin192  this script will not work : `flake8 manager setup.py`
we must have to specify `.py` file next to `flake8` : `flake8 setup.py`

Reply:
@ganeshhubale Ideally it should work

Reply:
@bhavin192 Done
I made changes.